src/Management/src/Endpoint/PublicAPI.Unshipped.txt

```diff
@@ -241,6 +241,9 @@ Steeltoe.Management.Endpoint.Actuators.HttpExchanges.HttpExchange.SerializedTime
 Steeltoe.Management.Endpoint.Actuators.HttpExchanges.HttpExchange.Session.get -> Steeltoe.Management.Endpoint.Actuators.HttpExchanges.HttpExchangeSession?
 Steeltoe.Management.Endpoint.Actuators.HttpExchanges.HttpExchange.Timestamp.get -> System.DateTime
 Steeltoe.Management.Endpoint.Actuators.HttpExchanges.HttpExchange.TimeTaken.get -> System.TimeSpan
+Steeltoe.Management.Endpoint.Actuators.HttpExchanges.HttpExchangePrincipal
+Steeltoe.Management.Endpoint.Actuators.HttpExchanges.HttpExchangePrincipal.HttpExchangePrincipal(string! name) -> void
+Steeltoe.Management.Endpoint.Actuators.HttpExchanges.HttpExchangePrincipal.Name.get -> string!

```

Comment:
Any idea why `HttpExchanges` is showing changes too? I'm confused

Reply:
The section only moved. I deleted the existing file and re-generated it.

Reply:
Regenerating the file explains it, though I would have hoped the contents would always be in the same order